src/components/requests/Requests.js

```diff
@@ -1,8 +1,15 @@
+/* eslint-disable array-callback-return */
+/* eslint-disable react/prop-types */
 import React from 'react';
 import './Requests.css';
+import { Link } from 'react-router-dom';
```

Comment:
Very fancy stuff on the linking here. All making sense ✅

Reply:
Yep looks very nice!